setup.cfg

```
@@ -22,7 +22,7 @@ classifiers =
 python_requires = >=3.7
 install_requires =
     prompt-toolkit~=3.0
-    typing_extensions~=3.7; python_version < '3.8'
+    typing_extensions>=3.7,<4.4; python_version < '3.8'
```

Expanding support version of `typing_extensions`.

---

```suggestion
    typing_extensions>=3.7,<5; python_version < '3.8'
```
`typing_extensions` now [follows semantic versioning](https://github.com/python/typing_extensions#overview). So it should be safe to allow `4.x`.